core/src/main/java/kafka/metrics/ClientMetricsReceiverPlugin.java

```diff
@@ -0,0 +1,67 @@
+/*
```

Comment:
This class is major overlap with the other open PR: https://github.com/apache/kafka/pull/14699.